test/if.civet

```diff
@@ -86,14 +86,45 @@ describe "if", ->
     if (x) y
   """
 
+  testCase """
+    if with multiple semicolon-separated statements
+    ---
+    if x
+      a; b
+    else
+      c; d
+    ---
+    if (x) {
+      a; b
+    }
+    else {
+      c; d
+    }
+  """
+
+  testCase """
+    if with multiple semicolon-separated commented statements
+    ---
+    if x
+      /*a*/ a; /*b*/ b
+    else
+      /*c*/ c; /*d*/ d
+    ---
+    if (x) {
+      /*a*/ a; /*b*/ b
+    }
+    else {
+      /*c*/ c; /*d*/ d
+    }
+  """
+
   // TODO
   testCase.skip """
```

Comment:
Should this still be `.skip`?

Reply:
It still doesn't work, but probably easyish to fix. Currently `if x then` compiles to `if (x)`, so it's missing either `;` or `{}`.